config/config.proto

```diff
@@ -34,6 +34,17 @@ message PushoverConfig {
   optional string user_key = 2;
 }
 
+message HipChatConfig {
+  // Hipchat auth token, https://www.hipchat.com/docs/api/auth
+  optional string auth_token = 1;
+  // Hipchat room id, https://www.hipchat.com/rooms/ids
+  optional int32 room_id = 2;
+  // color of message
+  optional string color = 3 [default = "red"];
```

Comment:
we can. but i guess i was assuming that these are alerts. want to change it to be be 'purple' and 'do not notify by default'?